scripts/build_awslambda.sh

```diff
@@ -5,7 +5,7 @@ get_current_pcm_version() {
   npm --prefix ./frontend pkg get version | tr -d '"'
```

Comment:
suggestion(non-blocking): can you please rename this script to something less harmful? It's called `build_awslambda.sh` but it also releases the image